### ISSUE TITLE
Set up similar to original UNet

### ISSUE THREAD

**Issue by povolann** [qubvel/segmentation_models#469]
Hello,

this is more for my understanding of the model, but I wondered if there is a way how I could implement the original UNet. Am I correct if I say that by setting BACKBONE = 'vgg16', I will get the same structure that was used for the original UNet, or not really?

**Comment by JordanMakesMaps** [comment]
Not quite; the (original paper)[https://arxiv.org/abs/1505.04597] uses a backbone similar to the VGG architecture (i.e. the same style), but it's not the exactly the same. This repo gives you the ability to use many different backbones either with or without pre-trained weights. If you wanted to get as close to the original U-NET as described in the paper, use VGG with no pre-trained weights. 

Another possibility is building something from scratch: see [this](https://keras.io/examples/vision/oxford_pets_image_segmentation/) keras tutorial

**Comment by povolann** [comment]
Thank you for the answer. I thought so, but I wasn't sure :)

**Comment by JordanMakesMaps** [comment]
There is also (this)[https://github.com/karolzak/keras-unet] repo, which does create the U-NET as described in the original paper, though I've found it doesn't achieve the same results as a model using this repo.
